Sources/SwiftSyntax/AbsolutePosition.swift

```diff
@@ -11,17 +11,17 @@
 //===----------------------------------------------------------------------===//
 
 /// An absolute position in a source file as text - the absolute utf8Offset from
-/// the start, line, and column.
-public struct AbsolutePosition {
+/// the start of the file.
+public struct AbsolutePosition: Comparable {
```

Comment:
I wonder if maybe we should just

```swift
typealias UTF8Offset = Int
```

and pass those around, instead of this wrapper struct, now that it's only got the one function. I could go either way.

Reply:
I considered removing the structs but I preferred to keep them for extra type-safety. They eliminate potential errors where you have some `Int` that may have come from anywhere (say a UTF16 count) and got passed as a UTF8 offset by mistake. The structs force you to be explicit in your handling/conversion of `Int`s.